03_Modules/Configuration/src/module/1.6/MessageApi.ts

```diff
@@ -128,6 +128,78 @@ export class ConfigurationOcpp16Api
     return Promise.all(confirmations);
   }
 
+  @AsMessageEndpoint(
+    OCPP1_6_CallAction.GetConfiguration,
+    OCPP1_6.GetConfigurationRequestSchema,
+  )
+  async getConfiguration(
+    identifier: string[],
+    tenantId: string,
+    request: OCPP1_6.GetConfigurationRequest,
+    callbackUrl?: string,
+  ): Promise<IMessageConfirmation[]> {
+    this._logger.debug('GetConfiguration request received:', request);
+
+    const confirmations = await Promise.all(
+      identifier.map(async (stationId) => {
+        const chargingStation =
+          await this._module.locationRepository.readChargingStationByStationId(
+            stationId,
+          );
+        if (!chargingStation) {
+          return {
+            success: false,
+            payload: `Charging station ${stationId} not found`,
+          };
+        }
+
+        const maxKeysConfig =
+          await this._module.changeConfigurationRepository.readOnlyOneByQuery({
+            where: {
+              stationId: stationId,
+              key: 'GetConfigurationMaxKeys',
+            },
+          });
+        const maxKeys = maxKeysConfig?.value
+          ? parseInt(maxKeysConfig.value, 10)
+          : Number.MAX_SAFE_INTEGER;
+        const keys = request.key || [];
+
+        const sendBatches = async (batches: string[][]) => {
+          return Promise.all(
+            batches.map(async (batch) => {
+              const correlationId = uuidv4();
+              await this._module.callMessageRepository.create(
+                CallMessage.build({ correlationId }),
+              );
+              return await this._module.sendCall(
```

Comment:
Can we find a way to add the batch index and the identifier into the confirmation payload? If anything is wrong, we can know which part exactly.